### PR TITLE
Add a line to example file

### PR DESCRIPTION
**Pull request by rikaimind-BuiMinhThu**

### CODE REVIEW

File: example.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 This is the example file of git study.
 This file's content is: We are awesome team.
+
+Example file of ThuBM
```
Comment (rikaimind-BuiMinhThu) [inline example.txt]: Please change to nakano sama

**Comment by hinodeharuki** [comment]
LGTM

**Comment by phuong1492** [comment]
⭕ K

**Comment by hatakazuhei** [comment]
lgtm

**Comment by jerrywdlee-kagebunshin** [comment]
lgtm
[LGTM](http://p3.pstatp.com/large/1e3000334d1876cb554)

**Pull request merged**